src/quest_manager/views.py

```diff
@@ -387,6 +387,11 @@ def quest_list(request, quest_id=None, template="quest_manager/quests.html"):
             available_quests = Quest.objects.get_available(request.user, remove_hidden)
         else:
             available_quests = Quest.objects.get_available_without_course(request.user)
+    
+    # this logic is in place as a temporary fix to issue #1162 (https://github.com/bytedeck/bytedeck/issues/1162)
+    # once a solution is found for the root cause of this issue, this line should be deleted.
+    # likely stems from PR #1158 (https://github.com/bytedeck/bytedeck/pull/1158)
+    available_quests = available_quests.order_by("sort_order", "-time_expired", "-date_expired", "name")
```

Comment:
Love it when code comments reference more details, thank you.

Reply:
Had an idea, what about using the model's ordering attribute here directly? That way we don't have it hard coded in 3 different places (model, here, and test)
I think you can go `available_quests.order_by(*Quest._meta.ordering)` or something like that... might have to treat the lists as args and expand with *args, not sure, just a guess

Reply:
hard-coding removed and replaced with *XPItem._meta.ordering